packages/libllvm/flang.subpackage.sh

```diff
@@ -0,0 +1,20 @@
+TERMUX_SUBPKG_INCLUDE="
+bin/f18
+bin/f18-parse-demo
+bin/tco
+bin/flang
+include/flang*
+lib/cmake/flang
+lib/libFortranCommon.a
+lib/libFortranCommon.a
+lib/libFortranEvaluate.a
+lib/libFortranEvaluate.a
+lib/lib/libFortranEvaluate.a
+lib/libFortranParser.a
+lib/libFortranSemantics.a
+lib/libFIROptimizer.a
+lib/libFortranRuntime.a
+"
+TERMUX_SUBPKG_DESCRIPTION="Fortran language frontend for LLVM"
+TERMUX_SUBPKG_BREAKS="libllvm (<< 11.0.0-1)"
+TERMUX_SUBPKG_REPLACES="libllvm (<< 11.0.0-1)"
```

Comment:
Should be no need to specify BREAKS and REPLACES as the subpackage is new, these two lines can be removed